.github/dependabot.yml

```diff
@@ -1,6 +1,33 @@
 version: 2
 updates:
+  # Maintain dependencies for the codebase
   - package-ecosystem: "gomod"
+    target-branch: "main"
+    directories:
+      - "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+    labels:
+      - "dependencies"
+    groups:
+      go_modules:
+        update-types:
+          - "minor"
+          - "patch"
+          - "major"
```

Comment:
Maybe it would be better to leave major updates as a separate PRs. There is a high chance that major updates will need code changes. WDYT ?

Reply:
I would rather start simply with all,
when it will really happen and we will see that it's needed then we can split it.